config/exporters/monitoring/grafana/base/prometheus-exporter-service.yaml

```diff
@@ -7,89 +9,100 @@ spec: {}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: exporter-sa
+  name: kube-rbac-proxy
   namespace: appstudio-grafana-datasource-exporter
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: exporter-cluster-role
+  name: kube-rbac-proxy2
 rules:
-- apiGroups: ['grafana.integreatly.org']
-  resources: ['grafanas']
-  verbs: ["get"]
+- apiGroups: ["authentication.k8s.io"]
+  resources:
+  - tokenreviews
+  verbs: ["create"]
+- apiGroups: ["authorization.k8s.io"]
+  resources:
+  - subjectaccessreviews
+  verbs: ["create"]
```

Comment:
What's the reason for changing this? Why do we need `create` now? Why don't we need `get` anymore?